packages/downloading-helpers/src/file-downloads/download-file.ts

```diff
@@ -89,23 +89,24 @@ export const downloadFile = async (
  *
  * Returns a list of the extracted files.
  *
- * Warning: this function is not thread safe. Do not try downloading a file to an extractPath that may already be in use by another process.
+ * Warning: this function is not thread safe. Do not try downloading a file to a `tmpZipFilePath`
```

Comment:
Do we also care about having a unique extract path? Haven't seen any thread safety warnings for the `extract` function, so I'm assuming no.

Reply:
The original warning is definitely valid too - that's the issue I'd hit in the past. `extract` will overwrite the file so if someone is using it when it runs they may get corrupt data.

Reply:
Ah ok good to know, I'll update the comment.

Just to clarify, was the issue that multiple gzips containing files with the same name were extracted into the same directory?

Reply:
Right, yeah. Specifically it was with the download of scripts from the snippet server (/S3 bucket) which always go to the same directory. There was a case where a script was downloaded inside replays so multiple replays would trigger the download simulatenously and overwrite each-other's script potentially while other replays were using it.